layouts/community/60_tsangan_hhkb/dohmain/readme.md

```diff
@@ -0,0 +1,37 @@
+# dohmain's 60% Tsangan HHKB layout
+## Layout
+![Layout Image](./layout.png)
```

Comment:
The Layout Image doesn't work. You should host the image elsewhere (like imgur, or a non-qmk_firmware repo) and direct link it here.

Reply:
imgur link added